schemas/1.0/general/business.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$id": "ministryofjustice/laa-criminal-legal-aid-schemas/main/schemas/1.0/general/business.json",
+  "title": "Business",
+  "description": "Holds attributes of a business",
+  "type": "object",
+  "businesses": {
+    "ownership_type": { "type": ["string"], "enum": ["applicant", "applicant_and_partner", "partner"] },
+    "business_type": { "type": ["string"], "enum": ["self_employed", "partnership", "director_or_shareholder"] },
+    "trading_name": { "type": ["string", "null"] },
+    "description": { "type": ["string", "null"] },
+    "trading_start_date": { "type": ["string", "null"], "format": "date" },
+    "address": { "type": ["object", "null"]},
+    "has_additional_owners": { "type": ["string", "null"], "enum": ["yes", "no", null] },
+    "additional_owners": { "type": ["string", "null"] },
+    "has_employees": { "type": ["string", "null"], "enum": ["yes", "no", null] },
+    "number_of_employees": { "type": ["integer", "null"] },
+    "salary": { "type": ["number", "null"] },
+    "total_income_share_sales": { "type": ["number", "null"] },
+    "percentage_profit_share": { "type": ["number", "null"] },
+    "turnover": { "type": ["object", "null"]},
```

Comment:
Not required, but the schema for turnover, drawings etc could be made more specific by referencing the definition for amount. Same could be done for address.